model/patch_lifecycle.go

```diff
@@ -428,6 +431,12 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		if err != nil {
 			return nil, errors.Wrapf(err, "error inserting parser project for version '%s'", patchVersion.Id)
 		}
+		if config != nil {
+			_, err = db.Collection(ProjectConfigsCollection).InsertOne(sessCtx, config)
+			if err != nil {
+				return nil, errors.Wrapf(err, "error inserting project configs for version '%s'", patchVersion.Id)
```

Comment:
nit: project config

Reply:
Renamed to project config for all mentions in the repo